dcos/package.py

```diff
@@ -845,7 +846,15 @@ def copy_to_cache(self, target_dir):
                 tmp_file = os.path.join(tmp_dir, 'packages.zip')
 
                 # Download the zip file.
-                urllib.request.urlretrieve(self.url, tmp_file)
+                requests.packages.urllib3.disable_warnings()
```

Comment:
We normally don't disable this warning in the `dcos` package instead we have been doing it at the entry point for each program in the `dcoscli` package. See: https://github.com/mesosphere/dcos-cli/blob/master/cli/dcoscli/config/main.py#L52